DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -5,15 +5,19 @@
    <version ClassVersion="11" checksum="834943006"/>
    <version ClassVersion="10" checksum="2474445311"/>
   </class>
-  <class name="SiTrackerGSRecHit2D" ClassVersion="11">
+  <class name="SiTrackerGSRecHit2D" ClassVersion="13">
+   <version ClassVersion="13" checksum="1916407081"/>
+   <version ClassVersion="12" checksum="1369340574"/>
```

Comment:
we should not skip versions due to some intermediate change
- remove version 12 and change 13 to 12